distribution/pull_v2.go

```diff
@@ -561,16 +589,15 @@ func (p *v2Puller) pullSchema2Layers(ctx context.Context, target distribution.De
 	defer cancel()
 
 	var (
-		downloadedRootFS *image.RootFS  // rootFS from registered layers
-		release          func()         // release resources from rootFS download
+		downloadedRootFS *image.RootFS // rootFS from registered layers
+		release          func()        // release resources from rootFS download
```

Comment:
This looks like diff noise

Reply:
I don't like this in the PR either, but this what `gofmt` says is right. It is just fixing formatting that used to be "wrong" before. I'd prefer keep it to "solve" the issue once and for all. Same for the other case below.

Reply:
@lmbarros Right, but it should probably be in a separate commit for formatting, so we're not grouping together unrelated changes. If, for example, we have to revert a change, we don't want to revert an unrelated but still good formatting change along with it.

Reply:
Yup, this is the right thing to do. I moved the formatting changes to a separate commit.